always add scroll-bar to UI to avoid jumping interface

bastibuck opened the pull request:
When switching between "Preset Configurations" and "Endpoints" in the management interface the whole container jumps to the left (maybe only on windows) as the scroll bar is added to the browser window when content overflows screen height.

![scroll-jump](https://user-images.githubusercontent.com/6306291/40888308-af1c7f36-6755-11e8-97e6-633336f4a600.gif)


This PR fixes this issue by always adding the scroll bar by default even when the content doesn't overflow the height.

pixelprodotco commented:
So the problem here is I would prefer no scroll over scroll if i can get it.  I believe height for the container is calculated wrong and pushing below the viewport height.  Instead of scrollbars everywhere (when theres no content to push it down) there should be scrollbars nowhere.

bastibuck commented:
Good idea as well! Will take a closer look why it's over 100vh high.Am 08.06.2018 02:51 schrieb James LeClair <notifications@github.com>:So the problem here is I would prefer no scroll over scroll if i can get it.  I believe height for the container is calculated wrong and pushing below the viewport height.  Instead of scrollbars everywhere (when theres no content to push it down) there should be scrollbars nowhere.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

bastibuck commented:
So I finally managed to take a closer look at this (work was busy, weather has been too amazing). You were missing the padding of the currounding container for your height calculation in **endpointStyles.js**. So I added the padding and also removed my older fix.

Should be fine and be good to go.

pixelprodotco commented on src/management-interface/src/Endpoints/endpointsStyles.js:
```diff
@@ -5,7 +5,7 @@ import { tabsTotalHeight } from '../Tabs/tabStyles'
 export const EndpointsContainer = glamorous.section({
   display: 'flex',
-  height: `calc(100vh - ${headerTotalHeight + tabsTotalHeight}px)`,
+  height: `calc(100vh - ${headerTotalHeight + tabsTotalHeight}px - 2rem)`,
```
the 2rem seems like an arbitrary value.  How did you come about with it?   We're ultimately taking the total height of the viewport, subtractig the header height, as well as the tabs height.  The result is being padded somehow (maybe there was margin added below the header or tabs or something.  That should be the value we need to equalize from.

bastibuck replied:
The 2rem comes from the surrounding container that has a padding of 1rem all around which makes up to 2rem height.

bastibuck replied:
https://github.com/pixelprodotco/appstrap/blob/master/src/management-interface/src/App.js#L20

pixelprodotco replied:
nice - ok good.
